appium/webdriver/extensions/search_context/android.py

```diff
@@ -28,7 +28,8 @@
 class AndroidSearchContext(BaseSearchContext):
     """Define search context for Android"""
 
-    def find_element_by_android_view_matcher(self, name=None, args=None, className=None):
+    def find_element_by_android_view_matcher(
+            self, name: Optional[str] = None, args: Any = None, className: Optional[str] = None) -> 'WebElement':
```

Comment:
[Notes] `args` can be various types(almost in str, List, Dict), so it's `Any`.

Reply:
if it is None by default, then Optional[Any] makes more sense